pyprof/prof/pointwise.py

```diff
@@ -145,13 +149,13 @@ def bytes_flops(self):
 
         # Unary
         if self.op() in Pointwise.unary + Pointwise.representation:
-            assert (len(self.input) == 1)
+            assert (len(self.input) >= 1)
             b = 2 * self.input[0].bytes
             f = self.input[0].size
 
         elif self.op() in Pointwise.exp_log + Pointwise.trig_trans + \
                 Pointwise.sqrt + Pointwise.error:
-            assert (len(self.input) == 1)
+            assert (len(self.input) >= 1)
```

Comment:
Reason for weakening the assertion?

Reply:
same as above.  I only make changes to assertions when I hit errors in the code.  This is a problem with the undocumented assumption that inputs are partitioned into named and unnamed arguments and only the unnamed ones are useful to these scripts

Reply:
Same as above. By definition arguments in python are partitioned into a list and a dictionary and we handle both cases. Both are extremely useful. See [conv.py](https://github.com/NVIDIA/PyProf/blob/master/pyprof/prof/conv.py#L82-L108) for a really complicated example of how `name` is used. Just because its mostly empty does not mean its open to abuse.